examples/profile.html

```diff
@@ -18,7 +22,10 @@
     </style>
   </head>
   <body ng-controller="MainController as ctrl">
-    <div class="profile" ngeo-profile="::ctrl.profileData"
+    <div id="map" ngeo-map="ctrl.map"></div>
+    <div class="profile"
+         ngeo-profile="ctrl.profile"
+         ngeo-profile-data="::ctrl.profileData"
```

Comment:
It would also be possible to add an `ngeo-profile-highlight` attribute and call `profile.highlight` in the directive.